plaidml/edsl/edsl.h

```diff
@@ -389,7 +413,7 @@ class IndexedTensor {
   ///
   /// Performs a multiplication combination within a contraction.
   ///
-  IndexedTensor operator*(const IndexedTensor& rhs) const { return IndexedTensor(PLAIDML_COMBO_OP_MUL, {*this, rhs}); }
+  IndexedTensor operator*(const IndexedTensor rhs) const { return IndexedTensor(PLAIDML_COMBO_OP_MUL, {*this, rhs}); }
```

Comment:
Was this intentional?

Reply:
Nope, artifact of an old change I undid; fixed